README.md

```diff
@@ -50,6 +50,11 @@ If you have locks - this will show you information about them.
 $ ./examples/show_lock_info.py -u http://192.168.1.161:3480/
 ~~~~
 
+Set a new door lock code
+~~~~
+$ ./examples/set_and_delete_door_code.py -u http://192.168.1.161:3480/ -n "John Doe" -p "12345678"
```

Comment:
The readme refers to `set_and_delete_door_code` - but the PR contains `delete_door_code` and `set_door_code`

Reply:
I will update the readme on this